salt/master.py

```diff
@@ -732,6 +732,13 @@ def __bind(self):
                     raise
                 # catch all other exceptions, so we don't go defunct
                 except Exception as exc:
+                    # since we are in an exceptional state, lets attempt to tell
+                    # the minion we have a problem, otherwise the minion will get
+                    # no response and be forced to wait for their max timeout
+                    try:
+                        socket.send('Unexpected Error in Mworker')
+                    except:  # pylint: disable=W0702
+                        pass
```

Comment:
This try/pass is not very nice. I would still add here at least debugging log entry, since this is an unexpected error and it must be at least be activated if somebody wants to have this in logs, once networking is debugged.